portfolio/src/main/webapp/comments/CommentFormRenderer.js

```diff
@@ -0,0 +1,33 @@
+class CommentFormRenderer extends RendererInterface {
+    constructor(targetElement) {
+        super();
+
+        this.targetElement = targetElement;
+    }
+
+    render(user) {
+        if(user) {
+            this.targetElement.innerHTML = `
+                <h2>New Comment</h2>
+                <form onsubmit="handlePostCommentFormSubmit(event)">
+                    <div class="comment-name-group">
+                        <label for="comment-name">Name:</label>
+                        <input name="comment-name" required />
```

Comment:
Why not to use email instead?

Reply:
Due to privacy issues. I though it would be better if the user can choose a nickname/name. Now, I could set up a unique nickname for each user, but this will take some time and I want to finish this ASAP! That's why I opted for enabling users to choose a name for each comment they post.